### PR TITLE
LACE-626 | Try to run tests using Python 3.9

### PR DESCRIPTION
**Pull request by macbre**
See https://github.com/bethinkpl/chronos/pull/119

### CODE REVIEW

**Comment by macbre** [comment]
```
nox > poetry run pylint src tests noxfile.py cli.py
************* Module src.pythink_toolbox.validating
src/pythink_toolbox/validating.py:59:30: E1136: Value 'Optional' is unsubscriptable (unsubscriptable-object)
src/pythink_toolbox/validating.py:110:10: E1136: Value 'Optional' is unsubscriptable (unsubscriptable-object)
************* Module src.pythink_toolbox.testing.parametrization
src/pythink_toolbox/testing/parametrization.py:7:0: E0239: Inheriting 'TypedDict', which is not a class. (inherit-non-class)
src/pythink_toolbox/testing/parametrization.py:7:0: R0903: Too few public methods (0/2) (too-few-public-methods)
src/pythink_toolbox/testing/parametrization.py:19:25: E1136: Value 'Union' is unsubscriptable (unsubscriptable-object)
src/pythink_toolbox/testing/parametrization.py:24:20: E1136: Value 'Union' is unsubscriptable (unsubscriptable-object)
************* Module tests.test_validating
tests/test_validating.py:14:12: E1136: Value 'Optional' is unsubscriptable (unsubscriptable-object)
tests/test_validating.py:17:15: E1136: Value 'Optional' is unsubscriptable (unsubscriptable-object)
tests/test_validating.py:12:0: R0903: Too few public methods (0/2) (too-few-public-methods)
tests/test_validating.py:75:12: E1136: Value 'Optional' is unsubscriptable (unsubscriptable-object)
tests/test_validating.py:78:15: E1136: Value 'Optional' is unsubscriptable (unsubscriptable-object)
tests/test_validating.py:91:0: R0903: Too few public methods (0/2) (too-few-public-methods)
tests/test_validating.py:132:0: R0903: Too few public methods (0/2) (too-few-public-methods)
************* Module tests.test_testing.test_paramatrization
tests/test_testing/test_paramatrization.py:9:0: R0903: Too few public methods (0/2) (too-few-public-methods)

-----------------------------------
Your code has been rated at 7.06/10
```

**Comment by miloszbednarzak** [comment]
@macbre by default we check python 3.9, but we don't require it to pass yet. We just check it to be sure if we are ready to switch to 3.9. But if we want to use 3.9 in Chronos we need pythink-toolbox to pass 3.9 checks also.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: [3.9]
```
Comment (miloszbednarzak) [inline .github/workflows/ci.yml:18]:
```suggestion
        python-version: [3.8, 3.9]
```
We want to support both versions.

**Comment by macbre** [comment]
The latest Pylint 2.6.0 still does not support Python 3.9 -> https://github.com/PyCQA/pylint/commit/2a86092204b123ff569965cbe609b243ee965181 is still in master.

The question is - do we want to wait for Pylint with Python 3.9 to be released or do we simply ignore pylint warnings in Python 3.9 build for now and tag a new version of `pythink-toolbox`? @miloszbednarzak ?

**Comment by miloszbednarzak** [comment]
@macbre I'd rather wait than ignore pylint. Pylint is our guardian angel, so I want to bet on it.
Also maintainers "promised" that release will be soon: https://github.com/PyCQA/pylint/issues/3882#issuecomment-754121011

**Comment by macbre** [comment]
https://github.com/PyCQA/pylint/milestone/24 - pylint's 2.6.1 milestone is only 41% complete now. We have to wait...

**Comment by macbre** [comment]
```
nox > poetry run pylint src tests noxfile.py cli.py
************* Module src.pythink_toolbox.testing.parametrization
src/pythink_toolbox/testing/parametrization.py:7:0: R0903: Too few public methods (0/2) (too-few-public-methods)
************* Module tests.test_validating
tests/test_validating.py:12:0: R0903: Too few public methods (0/2) (too-few-public-methods)
tests/test_validating.py:91:0: R0903: Too few public methods (0/2) (too-few-public-methods)
tests/test_validating.py:132:0: R0903: Too few public methods (0/2) (too-few-public-methods)
************* Module tests.test_testing.test_paramatrization
tests/test_testing/test_paramatrization.py:9:0: R0903: Too few public methods (0/2) (too-few-public-methods)
```

**Review by miloszbednarzak — APPROVED** [review]

**Pull request merged**